src/Microsoft.Azure.WebJobs.Host/Queues/QueueProcessor.cs

```diff
@@ -211,13 +213,9 @@ public QueueProcessor(QueueProcessorFactoryContext context)
         /// Called to raise the MessageAddedToPoisonQueue event
         /// </summary>
         /// <param name="e">The event arguments</param>
-        protected internal virtual void OnMessageAddedToPoisonQueue(EventArgs e)
+        protected internal virtual void OnMessageAddedToPoisonQueue(PoisonMessageEventArgs e)
```

Comment:
>OnMessageAddedToPoisonQueue [](start = 40, length = 27)

Why is this exposed?  
If somebody wants to react to the poison message, can they just use a [QueueTrigger] on the poison queue?